crates/sui-types/src/execution.rs

```diff
@@ -232,20 +232,6 @@ impl ExecutionTimeObservationKey {
             Command::Upgrade(_, _, _, _) => ExecutionTimeObservationKey::Upgrade,
         }
     }
-
-    // Returns the default estimated execution duration for the given key, for use if no
-    // observations are available.
-    pub fn default_duration(&self) -> Duration {
-        match self {
-            ExecutionTimeObservationKey::MoveEntryPoint { .. } => Duration::from_millis(1_500),
-            ExecutionTimeObservationKey::TransferObjects => Duration::from_millis(1),
-            ExecutionTimeObservationKey::SplitCoins => Duration::from_millis(1),
-            ExecutionTimeObservationKey::MergeCoins => Duration::from_millis(1),
-            ExecutionTimeObservationKey::Publish => Duration::from_millis(1),
-            ExecutionTimeObservationKey::MakeMoveVec => Duration::from_millis(1),
-            ExecutionTimeObservationKey::Upgrade => Duration::from_millis(1),
```

Comment:
the defaults for the native commands seem reasonable - a better approach might be to keep the old code but apply a max of 1.5s